src/main/config/simplified_tuning.h

```diff
@@ -23,7 +23,8 @@
 #include "flight/pid.h"
 #include "sensors/gyro.h"
 
-#define SIMPLIFIED_TUNING_MIN 0
+#define SIMPLIFIED_TUNING_PIDS_MIN 0
```

Comment:
```suggestion
#define SIMPLIFIED_TUNING_PIDS_MIN 50
```

Reply:
I think the reason this was changed to zero is that we want to allow setting PIDs (not filters) to zero.

Reply:
Yeah, my bad....

Reply:
I did not get why we need 50 here? @KarateBrot

Reply:
We don't. I'm just stupid... Your fix is fine.